### PR TITLE
Feat: add diagrams.net/draw.io embed

### LINKED ISSUE #820: Draw.io & Interactive SVG support
Hi

Outline looks absolutely fabulous.

I wanted to check if there was a plan to use draw.io to manage diagrams on outline. draw.io offers html5 and webdav integration

https://github.com/jgraph/drawio-html5

Also my project uses plantuml to create interactive svgs - so i can click on refs in one diagram to say go to another. I was wondering if outline supports/plans to support one of the following

a) native interactive svg support - when i upload svgs they are loaded as svgs into S3 but show up as a picture - i would expect them to be loaded inline so that the links work and content is searchable

b) plantuml support-  would be great to simply type plantuml in markdown to generate uml diagrams including the capability to including other plantuml content (constants/skins etc) from another location. I know gitlab has added this support

If so hopefully SVG+ draw.io support can be linked so that draw.io can be used to save and manage these. draw.io can create interactive svgs.

Otherwise really excited about Outline!

Bharat

### PR DESCRIPTION
**Pull request by wvds**
I went ahead and implemented the code to embed diagrams from Diagrams.net (formerly known as Draw.io).
You can get an embed link in a two different ways, both are working fine:
- File 🡒 Export as 🡒 URL...
- File 🡒 Publish 🡒 Link...

If there's a title in the URL it will also parse that and append it to the default title.
It was highly requested in our team, and it was also already mentioned in discussion #1408 and issue #820.

Let me know if there's anything that need to be changed or if there's other feedback!

closes #1408
closes #820

Edit: should be testable with this link:
`https://viewer.diagrams.net/?highlight=0000ff&edit=_blank&layers=1&nav=1&title=Outline%20test#R1VZNc5swEP01HN3hwyb20Tiu0zQfBzeTY0aFNSgVLBEi4P76LkEYMLaTgzOTznjG2rfLavV2n8BwFnG5kiyNbjEAYdhmUBrOpWHb1ti2jepnBtsamZpuDYSSBzqoBdb8L2jQ1GjOA8h6gQpRKJ72QR%2BTBHzVw5iUWPTDNij6u6YshAGw9pkYoo88UJE%2BhX3R4lfAw6jZ2XJntSdmTbA%2BSRaxAIsO5CwNZyERVb2KywWIiryGl%2Fq570e8u8IkJOojD7izyV1U%2FBrl1y%2B%2Ft9cP6VOarUY6yysTuT6wLlZtGwYoC5FNhldEXME6ZX7lKajfhEUqFmRZtGRZWndgw0ugTb1hhc12IBWUHUhXvAKMQckthWivPdPs6fFxGjaLthm7WYk6jdiBTA9AuMvdckQLTdNhyn7cWHAHV56yfk69x%2Fn9rf3wPJp8gLKAZkibKFWEISZMLFvUk5gnQUXSpUlWG3ODmGo6n0GprRYEyxX2yT7KbYa59OFE%2FY2KmAxBnYgb13HVWU52SoJgir%2F29XJ21g8Nqiuofm%2BDxECXfvclx8Yxyt4InFOAY6Zl66RVWP3f50rwBCpCIFNNTiqxTlsHDRpM06v6%2FciUxD%2BwQIGSkASTqssbLsQexAQPEzJ96hsQ7lVa4HTfzLUj5kEgjqmtPzZn0dfFt0lfYbPJQGFWg3UVNvksgTn%2FucDsocBO3b1fRGD2%2B6y3nFrvvw3OMJzTvbv%2F0GTaB%2B5%2B97Mmc%2Fz1OHKtfZKGL8hzkURm%2B7ny5ut89DnLfw%3D%3D`

### CODE REVIEW

**Comment by wvds** [comment]
Looks like the tests are failing on something different. The first time it ran it went fine and I didn't change anything except the commit message :)
https://app.circleci.com/pipelines/github/outline/outline/3830/workflows/dba69eb3-3e28-48df-80c6-4e5e4e85b590/jobs/6368

File: app/embeds/index.js [hunk 1]
```diff
@@ -115,6 +116,13 @@ export default [
     component: Gist,
     matcher: matcher(Gist),
   },
+  {
+    title: "Diagrams",
```
Comment (tommoor) [inline app/embeds/index.js:120]:
```suggestion
    title: "Diagrams.net",
```

Looks like "diagrams.net" is the name of the product

image

**Comment by tommoor** [comment]
Looks like the tests were flaky there 🤔  – thanks for the contribution!

**Review by tommoor — APPROVED** [review]
Works well, thanks 👍

**Comment by matbrgz** [comment]
Wesley could you please help me with others integration, some of them is open for a bounty? Please let me know if you are interested, emailing me, matbrgz@gmail.com.

**Pull request merged**